src/app/(main)/(pages)/watch-history/page.tsx

```diff
@@ -0,0 +1,115 @@
+import db from '@/db';
+import { getServerSession } from 'next-auth';
+import { authOptions } from '@/lib/auth';
+import { Content, CourseContent, VideoProgress } from '@prisma/client';
+import WatchHistoryClient from '@/components/WatchHistoryClient';
+import { Fragment } from 'react';
+
+export type TWatchHistory = VideoProgress & {
+  content: Content & {
+    parent: { id: number; courses: CourseContent[] } | null;
+    VideoMetadata: { duration: number | null } | null;
+  };
+};
+
+const formatWatchHistoryDate = (date: Date) => {
```

Comment:
don't we have dayjs? try to use dayjs and avoid creating these functions

Reply:
i'll make sure.